locust/web.py

```diff
@@ -174,6 +174,8 @@ def stop():
                 self._swarm_greenlet.kill(block=True)
                 self._swarm_greenlet = None
             environment.runner.stop()
+            if self.greenlet is not None:
+                self.greenlet.kill(block=True)
```

Comment:
Hmm... I'm worried this might cause issues with repeated gui runs. Have you tried that?

Reply:
What does meaning of `repeated gui runs`? I guess that different `--web-port` to run?

I have tried that it's normal to start again after sending the stop button, even for different ports.

Reply:
> I have tried that it's normal to start again after sending the stop button, even for different ports.

That's all I meant :) Thanks!

Reply:
Oops, it's my fault :neutral_face: for the issue #1995. Again, I understood the meaning of `with repeated gui runs`, run multiple tests in one running locust environment.